### PR TITLE
build from common parent dir /clients

### PR DESCRIPTION
**Pull request by octobocto**
We need builds for pretty much every package (except for the faucet).

Copying the build-$platform.sh scripts didn't seem like a good idea, so this is an attempt at reusing all the build scripts for all sub-packages.

### CODE REVIEW

File: .github/workflows/sidesail.yml [hunk 1]
```diff
@@ -42,9 +42,11 @@ jobs:
       # You can specify other versions if desired, see documentation here:
       # https://github.com/dart-lang/setup-dart/blob/main/README.md
       - uses: dart-lang/setup-dart@v1
-
       - name: Install dependencies
-        run: flutter pub get
+        run: |
+          cd sidesail
+          flutter pub get
+          cd ..
```
Comment (torkelrogstad) [inline .github/workflows/sidesail.yml:49]: `cd ..` does nothing here, as working directories are not persisted through tasks. And would be better with this instead IMO

```
run: flutter pub get
working-directory: sidesail
```

**Review by torkelrogstad — COMMENTED** [review]
Conceptually, this looks good. Completely untested, though

**Comment by torkelrogstad** [comment]
CI breakage looks unrelated. But apropos, we should be pinning a specific Git commit for the CUSF enforcer. Otherwise we'll get random breakages, depending on what the crabs over at `bip300301_enforcer` is up to

**Comment by octobocto** [comment]
> CI breakage looks unrelated. But apropos, we should be pinning a specific Git commit for the CUSF enforcer. Otherwise we'll get random breakages, depending on what the crabs over at `bip300301_enforcer` is up to

We should download binaries from releases.drivechain.info like we do for sidesail. It's there!
Screenshot 2024-10-07 at 14 22 52

File: .github/workflows/sidesail.yml (hunk 1 above)
Comment (octobocto) [reply .github/workflows/sidesail.yml:49]: thanks

**Pull request merged**